test/features/applicationmonitoring/label_version_detection.go

```diff
@@ -90,6 +90,14 @@ var (
 	}
 )
 
+/*
```

Comment:
The go way to comment multi lines is:
```
// Package path implements utility routines for manipulating slash-separated
// paths.
//
// The path package should only be used for paths separated by forward
// slashes, such as the paths in URLs. This package does not deal with
// Windows paths with drive letters or backslashes; to manipulate
// operating system paths, use the [path/filepath] package.
package path
```


could you please change it?

Reply:
Fixed in b969217d173736eb1cb514aa49225a78e49edb5c